docs/api-reference/io/xviz-binary-writer.md

```diff
@@ -0,0 +1,64 @@
+# XVIZBinaryWriters
+
+The class `XVIZBinaryWriter` will output the GLB based binary format of XVIZ data.
+
+## Example
+
+```js
+import {XVIZMetadataBuilder, XVIZBuilder} from '@xviz/builder';
+import {XVIZBinaryWriter, FileSink} from '@xviz/io';
+
+const sink = new FileSink('output-dir');
+const xvizWriter = new XVIZBinaryWriter(sink);
+
+const metadataBuilder = new XVIZMetadataBuilder();
+// build metadata
+xvizWriter.writeMetadata(metadataBuilder.getMetadata());
+
+const builder = new XVIZBuilder();
+for (let i = 0; i < 10; i++) {
+  // build frames
+  xvizWriter.writeFrame(i, builder.getFrame());
+}
+
+xvizWriter.close();
+```
+
+### Constructor
+
+```js
+import {FileSink, XVIZBinaryWriter} from '@xviz/io';
+
+const sink = new FileSink('output-dir');
+const xvizWriter = new XVIZBinaryWriter(sink);
+```
+
+Parameters:
+
+- `sink` (Object) - Object that manages writing data
+- `options.flatten` (Boolean) - flatten nested arrays
+
+### Methods
+
+##### writeMetadata(xvizMetadata)
+
+Encodes a log metadata to file.
+
+Parameters:
+
+- `xvizMetadata` (Object) - an XVIZ metadata object. See
+  [XVIZMetadataBuilder.getMetadata()](/docs/api-reference/xviz-metadata-builder.md#getMetadata).
+
+##### writeFrame(frameIndex, xvizFrame)
```

Comment:
Do we want to call this a frame still?  Since we don't use that term anywhere else?

Reply:
Do you have any suggestions?   Conceptually it is just a a sequence of data I am not tied to `Frame` in the API.

Reply:
#454 will address this once landed on master